services/github/src/handlers/job.js

```diff
@@ -237,6 +238,16 @@ async function jobHandler(message) {
     return await this.createExceptionComment({ debug, instGithub, organization, repository, sha, error: e });
   }
 
+  // only cancel previous tasks after we have successfully created new ones
+  // Cancel existing builds for non-default branches
+  if (graphConfig.autoCancelPreviousChecks !== false) {
```

Comment:
Curious as to why you're explicitly checking vs the `false` value here? Does this not work?

```js
if (graphConfig.autoCancelPreviousChecks) {
```

Reply:
yeah, so turns out it comes straight empty. Schema only validates but doesn't add default values to the missing props. 
So here we expect tc.yaml owner to explicitly say `false`, anything else is enabling this `true`, `undefined`